src/main/java/org/broadinstitute/hellbender/tools/sv/SVCallRecord.java

```diff
@@ -212,10 +212,6 @@ private static Pair<Boolean, Boolean> inferStrands(final GATKSVVCFConstants.Stru
             }
             return Pair.of(Boolean.FALSE, Boolean.TRUE);
         } else {
-            if (type.equals(GATKSVVCFConstants.StructuralVariantAnnotationType.INV)) {
```

Comment:
The javadoc comment for the method seems to be out of date now, can you update it?